### ISSUE TITLE
How to get the rendered HTML?

### ISSUE THREAD

**Issue by Crazy1van** [Tavenem/Blazor.MarkdownEditor#1]
I've been trying to find a way to get the HTML from the output.  Is there a way to do that?

**Comment by WilStead** [comment]
Not currently. If you would like to convert the source markdown to HTML, I personally recommend the [markdig](https://github.com/xoofx/markdig) library.

If your use case would be better handled by a WYSIWYG HTML editor, where the bound text is already HTML, I would recommend switching to the [Editor component](https://tavenem.com/Blazor.Framework/components/editor) in my newer [Blazor.Framework](https://github.com/Tavenem/Blazor.Framework) library. It supports a [WYSIWYG HTML mode](https://tavenem.com/Blazor.Framework/components/editor#heading-4) (as well as Markdown, if you still need support for that).

**Comment by Crazy1van** [comment]
Thanks.  Your MarkdownEditor fits my use case the best.  I just need a robust markdown editor with preview.  The alternative Blazor solutions out there are not as good imho.  In my case, I also wanted to be able to "Save" the resulting output to the client, hence my question about accessing the HTML. 

I figured it out though.  I set the "id" on the editor using the provided property, then used Javascript interop to get the container by id and its innerHTML.  Works well.
